sw/device/tests/entropy_src_fw_override_test.c

```diff
@@ -258,13 +270,60 @@ static status_t entropy_config(
   return OK_STATUS();
 }
 
+// Configure the entropy complex.
+static status_t reenable_entropy_src(
+    dif_entropy_src_single_bit_mode_t single_bit_mode,
+    bool bypass_conditioner) {
+  // Don't let the extract and insert interrupt handler run until we've enabled
+  // the entropy source.
+  irq_external_ctrl(false);
+
+  LOG_INFO("Disabling entropy_src");
+  // Disable the entropy source.
+  TRY(dif_entropy_src_stop(&entropy_src));
+  // Disable all health tests.
+  TRY(entropy_testutils_disable_health_tests(&entropy_src));
+
+  // Enable FW override.
+  TRY(dif_entropy_src_fw_override_configure(
+      &entropy_src,
+      (dif_entropy_src_fw_override_config_t){
+          .entropy_insert_enable = true,
+          .buffer_threshold = kEntropySrcFifoThreshold,
+      },
+      kDifToggleEnabled));
+
+  // Enable entropy_src.
+  LOG_INFO("Enabling entropy source");
+  TRY(dif_entropy_src_configure(
+      &entropy_src,
+      (dif_entropy_src_config_t){
+          .fips_enable = true,
+          .fips_flag = true,
+          .rng_fips = true,
+          .route_to_firmware = false,
+          .bypass_conditioner = bypass_conditioner,
+          .single_bit_mode = single_bit_mode,
+          .health_test_threshold_scope = false,
+          .health_test_window_size = kEntropySrcHealthTestWindowSize,
+          .alert_threshold = UINT16_MAX},
+      kDifToggleEnabled));
+
+  // Enable the interrupt handler to provide CSRNG with entropy.
+  irq_external_ctrl(true);
```

Comment:
I think this routine should be changed to:
1. First disable the entropy source.
2. Do some Ibex requests until a new seed is requested.
3. Wait for the CSRNG interrupt (`cs_entropy_req`).
4. Handle the interrupt by re-enabling ENTROPY_SRC and inject the seed.

Reply:
Okay, I'll change it.